### DISCUSSION TITLE
Allow players to select certain objects to still display when HUD is disabled

### DISCUSSION THREAD

**Discussion by mooshiros** [ppy/osu#29302] [Ideas]
In stable, even when disabling the HUD, you can still see the error bar at the bottom, which is very useful information to have. In lazer, the error bar disappears without the rest of the HUD. I was originally just going to ask for an option to keep the error bar on, but I think it would make more sense to allow players to select certain objects (maybe the button counter, or the accuracy, etc) to be exempted from HUD disabling.

**Comment by peppy** [comment]
This is planned!
